src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/Formatting/FormattingHandler.ts

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { RequestType } from 'vscode-languageclient';
+import { RazorDocumentManager } from '../RazorDocumentManager';
+import { RazorLanguageServerClient } from '../RazorLanguageServerClient';
+import { RazorLogger } from '../RazorLogger';
+import { convertTextEditToSerializable, SerializableTextEdit } from '../RPC/SerializableTextEdit';
+import { SerializableFormattingParams } from './SerializableFormattingParams';
+import { SerializableFormattingResponse } from './SerializableFormattingResponse';
+
+export class FormattingHandler {
+    private static readonly provideFormattingEndpoint = 'textDocument/formatting';
+    private formattingRequestType: RequestType<SerializableFormattingParams, SerializableFormattingResponse, any> = new RequestType(FormattingHandler.provideFormattingEndpoint);
+    private emptyFormattingResponse = new SerializableFormattingResponse();
+
+    constructor(
+        private readonly documentManager: RazorDocumentManager,
+        private readonly serverClient: RazorLanguageServerClient,
+        private readonly logger: RazorLogger) {
+    }
```

Comment:
The fact that TSLint recommends this instead of `{ }` offends me, but I understand the need to  appease our overlords.

Reply:
I tested and turns out TSLint actually doesn't care if we do `{ }`. I did it the original way because that's how it was done in pre-existing classes. Pushing a change now that should change this 😄